README.md

```diff
@@ -4,7 +4,13 @@ The Coiled Runtime is a conda metapackage which makes it easy to get started wit
 
 ## Install
 
-`coiled-runtime` can be installed with:
+Currently, `coiled-runtime` is not in the default channel. The conda-forge channel must be enabled:
```

Comment:
I'm not sure this is actually why you ran into trouble in https://github.com/coiled/coiled-runtime/issues/21.

If we look at your mamba output, it found `coiled-runtime` just fine (from the `coiled` channel, as specified), but didn't find `coiled`.

Reply:
yes because `coiled-runtime` is in the coiled channel, but he didn't have conda-forge set up as a channel so when it's looking for coiled it's looking only in the default channel hence the problem.